### PR TITLE
BaSP-MR-107: Implementation of shared Components into Form.

### PR DESCRIPTION
**Pull request by IvanJukonis**
Here is my pull request regarding ticket number 107, where the implementation of global components was required within the SuperAdmins form.

**IMPORTANT**: Since I am responsible for both components (table and form), the first commit is a copy of the PR related to ticket 106. This was done because I am not allowed to create a branch from another branch, and I needed the previous work to resolve this ticket. Since I am handling both PRs, any conflicts arising from this will be resolved promptly.

Therefore, I recommend not making any corrections to the changes made in the first commit.

### CODE REVIEW

**Comment by IvanJukonis** [comment]
[SUPERADMINS - Form](https://trello.com/c/hPIGpFzo/107-superadmins-form)

File: src/Components/SuperAdmins/Form/index.jsx [hunk 1]
```diff
@@ -1,78 +1,126 @@
-import React, { useState } from 'react';
-import { ModalConfirm } from '../../Shared';
+import React, { useState, useEffect } from 'react';
+import { ModalConfirm, Inputs, Button, ModalSuccess } from '../../Shared';
+import { useLocation } from 'react-router-dom';
+import styles from './form.module.css';
+import { useHistory, useParams } from 'react-router-dom/cjs/react-router-dom.min';
```
Comment (FrancoLelli) [inline src/Components/SuperAdmins/Form/index.jsx:5]:
```suggestion

```

File: src/Components/SuperAdmins/Form/index.jsx [hunk 2]
```diff
@@ -1,78 +1,126 @@
-import React, { useState } from 'react';
-import { ModalConfirm } from '../../Shared';
+import React, { useState, useEffect } from 'react';
+import { ModalConfirm, Inputs, Button, ModalSuccess } from '../../Shared';
+import { useLocation } from 'react-router-dom';
```
Comment (FrancoLelli) [inline src/Components/SuperAdmins/Form/index.jsx:3]:
```suggestion
import { useLocation, useHistory, useParams } from 'react-router-dom';
```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Nice Job, but resolve that comments!

File: package.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
     ]
   },
   "engines": {
-    "node": "18.x",
+    "node": "16.8.0",
```
Comment (FrancoLelli) [inline package.json:42]: set 18.x version

**Review by FrancoLelli — CHANGES_REQUESTED** [review]

**Review by FrancoLelli — APPROVED** [review]

**Review by maurojjzz — APPROVED** [review]
good job

**Comment by IvanJukonis** [comment]
[[BUG] Super Admin - The list is not displayed](https://trello.com/c/jCURDTfk/146-bug-super-admin-the-list-is-not-displayed)

**Pull request merged**